go-app-ussd_chw.js

```diff
@@ -778,29 +821,65 @@ go.app = function() {
                     new Choice('ven_ZA', 'Tshivenda'),
                     new Choice('nbl_ZA', 'isiNdebele'),
                 ],
-                next: "state_save_subscription"
+                next: "state_set_registration_type"
+            });
+        });
+
+        self.add("state_set_registration_type", function(name){
+            /*var registration_types = {
+                'sms': 'SMS',
+                'whatsapp': 'WhatsApp',
+            };*/
```

Comment:
We can probably remove this commented out code

Reply:
Thanks for pointing this out